clients/web/src/views/body/FolderView.js

```diff
@@ -19,7 +19,12 @@ var FolderView = View.extend({
         this.folderEdit = settings.folderEdit || false;
         this.itemCreate = settings.itemCreate || false;
 
+        this.render();
+    },
+
+    render: function () {
         this.hierarchyWidget = new HierarchyWidget({
```

Comment:
In the case of `FolderView` here, it doesn't really matter at all. Since it's a body view, it's expected to self-render, so we could move the instantiation of `HierarchyWidget` up to the end of `FolderView.initialize` and remove the `FolderView.render` method entirely.

I put it here only for consistency with the `CollectionView` and `UserView`, where the location matters more, as described in this PR's description.

Honestly, I don't yet have a totally clean vision that I'd like to advocate for how and when child widgets get instantiated and rendered, but this is an improvement in efficiency over the old code, and fixes a broken test.